room.h

```diff
@@ -174,6 +174,15 @@ namespace QMatrixClient
             /** Mark all messages in the room as read */
             void markAllMessagesAsRead();
 
+            void inviteCall(const QString& callId, const int& lifetime,
+                            const QString& sdp);
+            void callCandidates(const QString& callId,
+                                const QJsonArray& candidates);
```

Comment:
As mentioned above, please try to not use `QJsonObject` unless you actually intend to pass some opaque JSON object. In this particular case, CS API is well-defined on the object contents so just pass a `QVector` of supplementary structures.

Reply:
Using QJsonArray makes it easier to pass to directly to WebRTC handler which is written JavaScript, since all WebRTC will probably be handled by a browser (qwebengine or oxide) this makes sense.

Reply:
Oh, I see, didn't know about that. Then sure, please keep it as is.

Reply:
Would be great to see doc comments next to the declarations, by the way :) I'm not very consistent at putting them around, but 0.2 has an explicit goal about documenting the whole public API of the library, so these comments will have to arrive sooner or later.